src/big/big_rational.cr

```diff
@@ -360,6 +360,13 @@ struct BigRational < Number
     to_s io
   end
 
+  # :ditto:
+  def format(io : IO, separator = '.', delimiter = ',', decimal_places : Int? = nil, *, group : Int = 3, only_significant : Bool = false) : Nil
+    denominator = self.denominator.format(separator, delimiter, decimal_places, group: group, only_significant: only_significant)
+    numerator = self.numerator.format(separator, delimiter, decimal_places, group: group, only_significant: only_significant)
```

Comment:
Each of these could also format to the `io` directly without allocating an intermediary string